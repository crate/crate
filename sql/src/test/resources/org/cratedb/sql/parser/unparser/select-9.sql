SELECT X + $2, Z FROM T1 WHERE Y <> $1
