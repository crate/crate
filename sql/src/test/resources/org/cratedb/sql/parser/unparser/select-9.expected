SELECT (x + $2), z FROM t1 WHERE y <> $1